🌐 Add German translation for `docs/de/docs/tutorial/path-params.md`

nilslindemann opened the pull request:
There is an accidental overlap with #10281 by DevSpace88.

← `async.md` (#10449)
→ `tutorial/query-params.md` (#10293)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
German translation of docs/tutorial/path-params.md

* Full translation
* In sync with current en/docs/tutorial/path-params.md
* Hand-translated (Google translate rarely used)
* Checked for errors using LanguageTool
* Manually checked

tiangolo commented:
📝 Docs preview for commit 0252f0b2e433a3bb647fbfba335741962b2bceaf at: https://8488bdff.fastapitiangolo.pages.dev

nilslindemann commented:
@tiangolo The German translation for the specific page is missing from most of these commit previews. I am sorry if I missed something. Do I have to update any global index file?

DevSpace88 commented:
That's because you forgot to put path-params.md in the directory ```tutorial```
In your commit your path is  ```docs/de/docs/path-params.md```
it should be:
```docs/de/docs/tutorial/path-params.md```

But, I actually did this exact [PR](https://github.com/tiangolo/fastapi/pull/10281)  yesterday evening or tonight. I also noticed, that you did other PR's that were for files that also already had a PR. 
Please check the open [PR's](https://github.com/tiangolo/fastapi/pulls) before starting a new one.

Commit:
Put file into the correct location

tiangolo commented:
📝 Docs preview for commit 3809f5beab9741b9ca4f16f2171bd74694fcdcb3 at: https://aa1ff140.fastapitiangolo.pages.dev

nilslindemann commented:
> @nilslindemann That's because you forgot to put path-params.md in the directory `tutorial` In your commit your path is `docs/de/docs/path-params.md` it should be: `docs/de/docs/tutorial/path-params.md`

Sorry, my mistake. Fixed.

> But, I actually did this exact [PR](https://github.com/tiangolo/fastapi/pull/10281) yesterday evening or tonight. I also noticed, that you did other PR's that were for files that also already had a PR. Please check the open [PR's](https://github.com/tiangolo/fastapi/pulls) before starting a new one.

I checked all PRs on Monday (23-9-18). Unfortunately, that was before you made your PR. Shit :-(

**I will from now on post [here](https://github.com/tiangolo/fastapi/discussions/9203?sort=old) before I start a new translation.** I would be grateful if others could do so too, so that this does not happen anymore.

Regarding the other commits, there were a few errors, and some translations were not in sync or just partially translated. The authors did not respond in their PRs. So I decided to take them as base and create a bunch of new PRs.

I am planning to go through the whole docs, document by document. If you are interested, we can join our forces. I have invited you as a collaborator in my [workbranch](https://github.com/nilslindemann/fastapi/tree/deutsche-uebersetzung-arbeitsbranch).

DevSpace88 commented:
> **I will from now on post [here](https://github.com/tiangolo/fastapi/discussions/9203?sort=old) before I start a new translation.** I would be grateful if others could do so too, so that this does not happen anymore.

Sounds like a good idea. I will do the same then.

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+# Pfad-Parameter
+
+Sie können Pfad "Parameter" oder - "Variablen" mit der gleichen Syntax deklarieren, welche in Python <abbr title="Format String – Formatierte Zeichenkette: Die Zeichenkette enthält Variablen, die mit geschweiften Klammern umschlossen sind. Solche Stellen werden durch den Wert der Variable ersetzt">Format Strings</abbr> verwendet wird:
```
```suggestion
Sie können Pfad-"Parameter" oder -"Variablen" mit der gleichen Syntax deklarieren, welche in Python <abbr title="Format String – Formatierte Zeichenkette: Die Zeichenkette enthält Variablen, die mit geschweiften Klammern umschlossen sind. Solche Stellen werden durch den Wert der Variable ersetzt">Format Strings</abbr> verwendet wird:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+## Pfad-Parameter mit Typen
+
+Sie können den Typen eines Pfad-Parameters in der Argumentliste der Funktion deklarieren, mittels Standard Python Typ Annotationen:
```
```suggestion
Sie können den Typen eines Pfad-Parameters in der Argumentliste der Funktion deklarieren, mit den normalen Typ-Annotationen in Python:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+In diesem Fall wird `item_id` als `int` deklariert, also als Ganzzahl.
+
+!!! check
+    Dadurch erhalten Sie Editor Unterstützung innerhalb Ihrer Funktion, mit Fehlerprüfungen, Codevervollständigung, usw.
```
```suggestion
    Dadurch erhalten Sie Editor-Unterstützung innerhalb Ihrer Funktion, mit Fehlerprüfungen, Codevervollständigung, usw.
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+!!! check
+    Beachten Sie, dass der Wert, den Ihre Funktion erhält und zurückgibt, die Zahl `3` ist, also ein `int`. Nicht die Zeichenkette `"3"`, also ein `str`.
+
+    Sprich, mit dieser Typ Deklaration wird **FastAPI** die Anfrage automatisch <abbr title="Die Zeichenkette, die von einer HTTP Anfrage kommt, nach Python Daten konvertieren">"parsen"</abbr>.
```
```suggestion
    Sprich, mit dieser Typ-Deklaration wird **FastAPI** die Anfrage automatisch <abbr title="Die Zeichenkette, die von einer HTTP Anfrage kommt, in Python-Objekte konvertieren">"parsen"</abbr>.
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+## Daten-Validierung
+
+Aber wenn Sie im Browser <a href="http://127.0.0.1:8000/items/foo" class="external-link" target="_blank">http://127.0.0.1:8000/items/foo</a> besuchen, erhalten Sie eine hübsche HTTP-Fehlermeldung:
```
```suggestion
Wenn Sie aber im Browser <a href="http://127.0.0.1:8000/items/foo" class="external-link" target="_blank">http://127.0.0.1:8000/items/foo</a> besuchen, erhalten Sie eine hübsche HTTP-Fehlermeldung:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+Der Pfad-Parameter `item_id` hatte den Wert `"foo"`, was kein `int` ist.
+
+Die gleiche Fehlermeldung würde angezeigt werden, wenn Sie ein `float` (eine Fließkommazahl) statt eines `int`s übergeben würden, wie etwa in: <a href="http://127.0.0.1:8000/items/4.2" class="external-link" target="_blank">http://127.0.0.1:8000/items/4.2</a>
```
```suggestion
Die gleiche Fehlermeldung würde angezeigt werden, wenn Sie ein `float` (also eine Kommazahl) statt eines `int`s übergeben würden, wie etwa in: <a href="http://127.0.0.1:8000/items/4.2" class="external-link" target="_blank">http://127.0.0.1:8000/items/4.2</a>
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+## Dokumentation
+
+Wenn Sie Ihren Browser unter <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a> öffnen, sehen Sie eine automatisch generierte, interaktive API Dokumentation:
```
```suggestion
Wenn Sie die Seite <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a> in Ihrem Browser öffnen, sehen Sie eine automatisch generierte, interaktive API-Dokumentation:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+<img src="/img/tutorial/path-params/image01.png">
+
+!!! check
+    Wiederum, mit dieser gleichen Python Typ Deklaration gibt Ihnen **FastAPI** eine automatisch generierte, interaktive Dokumentation (verwendet die Swagger Benutzeroberfläche).
```
```suggestion
    Wiederum, mit dieser gleichen Python Typ Deklaration gibt Ihnen **FastAPI** eine automatisch generierte, interaktive Dokumentation (verwendet die Swagger-Benutzeroberfläche).
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+## Nützliche Standards. Alternative Dokumentation
+
+Und weil das generierte Schema vom <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md" class="external-link" target="_blank">OpenAPI</a> Standard kommt, gibt es viele kompatible Werkzeuge.
```
```suggestion
Und weil das generierte Schema vom <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md" class="external-link" target="_blank">OpenAPI</a>-Standard kommt, gibt es viele kompatible Werkzeuge.
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+Und weil das generierte Schema vom <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md" class="external-link" target="_blank">OpenAPI</a> Standard kommt, gibt es viele kompatible Werkzeuge.
+
+Zum Beispiel bietet **FastAPI** selbst eine alternative API Dokumentation (verwendet ReDoc), welche Sie unter <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a> einsehen können:
```
```suggestion
Zum Beispiel bietet **FastAPI** selbst eine alternative API-Dokumentation (verwendet ReDoc), welche Sie unter <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a> einsehen können:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+<img src="/img/tutorial/path-params/image02.png">
+
+Auf dem gleichen Weg gibt es viele kompatible Werkzeuge. Inklusive Code generierenden Werkzeugen für viele Sprachen.
```
```suggestion
Auf dem gleichen Weg gibt es viele kompatible Werkzeuge. Inklusive Codegenerierung für viele Sprachen.
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+Und Sie haben auch einen Pfad `/users/{user_id}`, um Daten über einen spezifischen Nutzer zu erhalten, mittels einer Nutzer-ID.
+
+Weil *Pfad-Operationen* in Reihenfolge ausgewertet werden, müssen Sie sicherstellen, dass der Pfad `/users/me` vor `/users/{user_id}` deklariert wurde:
```
```suggestion
Weil *Pfad-Operationen* in ihrer Reihenfolge ausgewertet werden, müssen Sie sicherstellen, dass der Pfad `/users/me` vor `/users/{user_id}` deklariert wurde:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+## Vordefinierte Parameterwerte
+
+Wenn Sie eine *Pfad-Operation* haben, welche einen *Pfad-Parameter* hat, aber Sie wollen, dass dessen gültige Werte vordefiniert sind, können Sie ein Standard Python <abbr title="Enumeration – Aufzählung">`Enum`</abbr> verwenden.
```
```suggestion
Wenn Sie eine *Pfad-Operation* haben, welche einen *Pfad-Parameter* hat, aber Sie wollen, dass dessen gültige Werte vordefiniert sind, können Sie ein normales Python-<abbr title="Enumeration – Aufzählung">`Enum`</abbr> verwenden.
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+### Erstellen Sie eine `Enum`-Klasse
+
+Importieren Sie `Enum` und erstellen Sie eine Kind-Klasse, die von `str` und `Enum` erbt.
```
```suggestion
Importieren Sie `Enum` und erstellen Sie eine Unterklasse, die von `str` und `Enum` erbt.
```

nilslindemann replied:
Kind-Klasse gefällt mir, schon philosophisch, besser.

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+### Mit Python *<abbr title="Enumeration – Aufzählung">Enumerationen</abbr>* arbeiten
+
+Der *Pfad-Parameter* wird ein *Mitglied einer Enumeration* sein.
```
```suggestion
Der *Pfad-Parameter* wird ein *Member einer Enumeration* sein.
```

Ich denke, wenn wir das ohnehin als Begriff einführen, können wir auch gleich den englischen Namen verwenden.

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+#### *Enumerations-Mitglieder* vergleichen
+
+Sie können ihn mit einem Mitglied Ihres Enums `ModelName` vergleichen:
```
```suggestion
Sie können ihn mit einem Member Ihres Enums `ModelName` vergleichen:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+#### Den *Enumerations-Wert* erhalten
+
+Den tatsächlichen Wert (in diesem Fall ein `str`) erhalten Sie via `model_name.value`, oder generell, `ihr_enum_mitglied.value`:
```
```suggestion
Den tatsächlichen Wert (in diesem Fall ein `str`) erhalten Sie via `model_name.value`, oder generell, `ihr_enum_member.value`:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+#### *Enumerations-Mitglieder* zurückgeben
+
+Sie können *Enum-Mitglieder* in ihrer *Pfad-Operation* zurückgeben, sogar verschachtelt in einem JSON-Körper (z.B. als `dict`).
```
```suggestion
Sie können *Enum-Member* in ihrer *Pfad-Operation* zurückgeben, sogar verschachtelt in einem JSON-Body (z.B. als `dict`).
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+Sie können *Enum-Mitglieder* in ihrer *Pfad-Operation* zurückgeben, sogar verschachtelt in einem JSON-Körper (z.B. als `dict`).
+
+Diese werden zu ihren entsprechenden Werten konvertiert (in diesem Fall Zeichenketten), bevor sie zum Klient übertragen werden:
```
```suggestion
Diese werden zu ihren entsprechenden Werten konvertiert (in diesem Fall Zeichenketten), bevor sie zum Client übertragen werden:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
````diff
@@ -0,0 +1,252 @@
+{!../../../docs_src/path_params/tutorial005.py!}
+```
+
+In Ihrem Klienten erhalten Sie eine JSON-Antwort, wie etwa:
````
```suggestion
In Ihrem Client erhalten Sie eine JSON-Antwort, wie etwa:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+### Pfad Konverter
+
+Mittels einer Option direkt von Starlette können Sie einen *Pfad Parameter* deklarieren, der einen Pfad enthalten soll, indem Sie eine URL wie folgt definieren:
```
```suggestion
Mittels einer Option direkt von Starlette können Sie einen *Pfad-Parameter* deklarieren, der einen Pfad enthalten soll, indem Sie eine URL wie folgt definieren:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+## Rekapitulation
+
+In **FastAPI**, mittels kurzer, intuitiver Typ-Deklarationen, erhalten Sie:
```
```suggestion
In **FastAPI** erhalten Sie mittels kurzer, intuitiver Typ-Deklarationen:
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+In **FastAPI**, mittels kurzer, intuitiver Typ-Deklarationen, erhalten Sie:
+
+* Editor Unterstützung: Fehlerprüfungen, Codevervollständigung, usw.
```
```suggestion
* Editor-Unterstützung: Fehlerprüfungen, Codevervollständigung, usw.
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+* Editor Unterstützung: Fehlerprüfungen, Codevervollständigung, usw.
+* Daten "<abbr title="Die Zeichenkette, die von einer HTTP Anfrage kommt, nach Python Daten konvertieren">parsen</abbr>"
+* Daten-Validierung
+* API Annotationen und automatische Dokumentation
```
```suggestion
* API-Annotationen und automatische Dokumentation
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,252 @@
+
+Und Sie müssen sie nur einmal deklarieren.
+
+Das ist wahrscheinlich der sichtbarste Unterschied zwischen **FastAPI** und alternativen Frameworks (abgesehen von der reinen Performanz).
```
```suggestion
Das ist wahrscheinlich der sichtbarste Unterschied zwischen **FastAPI** und alternativen Frameworks (abgesehen von der reinen Performance).
```

GeorchW commented on docs/de/docs/tutorial/path-params.md:
````diff
@@ -0,0 +1,252 @@
+{!../../../docs_src/path_params/tutorial001.py!}
+```
+
+Der Wert des Pfad-Parameters `item_id` wird ihrer Funktion als das Argument `item_id` übergeben.
````
```suggestion
Der Wert des Pfad-Parameters `item_id` wird Ihrer Funktion als das Argument `item_id` übergeben.
```

GeorchW reviewed (changes requested):
Nett!

Zwei Dinge, die sich durch den Text durchziehen:
1. Ich würde etwas weniger aggressiv eindeutschen. Z.B. die Übersetzungen "performance" => "Performanz" und "client" => "Klient" lassen einen als Deutschen eher stolpern. Ich bin mir recht sicher, dass ich in deutschsprachigen IT-Magazinen auch keine Übersetzungen dafür finden würde.
2. Komposita werden im Deutschen grundsätzlich nicht mit Leerzeichen geschrieben, sondern entweder am Stück oder mit Bindestrich, z.B. "Pfad-Parameter" oder "Pfadparameter", aber nicht "Pfad Parameter".

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

tiangolo commented:
📝 Docs preview for commit 3495486c605d629ff3fdeca0e858c295a615f7be at: https://09da0811.fastapitiangolo.pages.dev

nilslindemann commented:
Hallo @GeorchW,

Danke für deine vielen guten Vorschläge.

> 2. Komposita werden im Deutschen grundsätzlich nicht mit Leerzeichen geschrieben, sondern entweder am Stück oder mit Bindestrich, z.B. "Pfad-Parameter" oder "Pfadparameter", aber nicht "Pfad Parameter".

Da stimme ich zu. Allerdings scheint das seit der neuen Rechtschreibung optional zu sein, zumindest hat LanguageTool nicht genörgelt. Deswegen ist es mir auch bei der Rechtschreibprüfung durch die Finger geflutscht. In späteren Übersetzungen verwende ich, wenn möglich (hoffentlich) den Bindestrich. Danke für diese Korrekturen.

> 1. Ich würde etwas weniger aggressiv eindeutschen. Z.B. die Übersetzungen "performance" => "Performanz" und "client" => "Klient" lassen einen als Deutschen eher stolpern. Ich bin mir recht sicher, dass ich in deutschsprachigen IT-Magazinen auch keine Übersetzungen dafür finden würde.

Ja, das ist ein schwieriges Problem, was Übersetzen und was nicht. Ich übersetze mittlerweile auch weniger (beispielsweise belasse ich "Request-Body" und "Response-Body") und füge notfalls eine deutsche Übersetzung in Klammern oder als Hover-Info hinzu. Bin hier wohl zu weit gegangen.

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Fix

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

Commit:
+GeorchW Vorschlag

Co-authored-by: Georg Wicke-Arndt <g.wicke-arndt@outlook.com>

tiangolo commented:
📝 Docs preview for commit e9baaf6ff64ed88c628fe1febafa1b05ff5428c0 at: https://be6b5bd0.fastapitiangolo.pages.dev

nilslindemann commented:
Auch hier vielen Dank für das Review, @GeorchW!

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit 19a6b8cff569ca6d810d139a61b0526e7d830702 at: https://ac5c0ef3.fastapitiangolo.pages.dev

Commit:
Clarify Enum explanation

tiangolo commented:
📝 Docs preview for commit 94b9d7288fcb9d5be7bc2384ff092894442c1e67 at: https://e496d152.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit cbb3bd5da7b17120edce2f6e132ffec06840671d at: https://f7d2588b.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit 9ce41eb3257e4ba6e89ffe69a748cba773161830 at: https://7cd6099b.fastapitiangolo.pages.dev

Commit:
"Kindklasse" ohne Bindestrich

tiangolo commented:
📝 Docs preview for commit 75039d9bd0a1f964b42e96c18742703a3e2f802d at: https://5fe9b3b4.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit 09663bfd419bf5940d116fa734f7a7360f0c0d4f at: https://7b9e2f9a.fastapitiangolo.pages.dev

Commit:
Improve translation of "path operation"

tiangolo commented:
📝 Docs preview for commit aded33c6d8f4f671a7cbbd6ecf54418877450b6e at: https://8ed79103.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit 9877b60c1a8575bd38ead78f77912039c0b6cd7e at: https://39c8ba41.fastapitiangolo.pages.dev

Commit:
A few improvements

tiangolo commented:
📝 Docs preview for commit 5a5cd418862afe99ae86b78de3886733243ded0a at: https://87d86527.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit 5d762315f5b24fbcad89883f47dfd8414be8e3df at: https://2cfdaec1.fastapitiangolo.pages.dev

Commit:
"Performance" -> "Performanz"

Changed my opinion. https://de.wikipedia.org/wiki/Rechenleistung uses "Performanz", so I use it too.

tiangolo commented:
📝 Docs preview for commit 9b2ecaf749b34ef8e35cd4dc3899d436d48fa0de at: https://d9ad5278.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit e2401db32ae4590a9860a9047ab016727a186545 at: https://3ac99c13.fastapitiangolo.pages.dev

Commit:
"Normal" -> "Standard"

tiangolo commented:
📝 Docs preview for commit 32caf4d1b0221691a41f2330eea5ad15c5253828 at: https://f8f184fa.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit b51bb2655fef1528386ae979e271db3088cdb642 at: https://01cdbfb0.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

Commit:
Sync with #10043

Commit:
"Tip" -> "Tipp"

tiangolo commented:
📝 Docs preview for commit b1a4077982c03741d67be59d6253408ba1c586d5 at: https://78ca0b12.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit 0be5693a237f611c35783012b59ec2c530b33e89 at: https://9bb2c91b.fastapitiangolo.pages.dev

Commit:
"Nutzer" -> "Benutzer"

tiangolo commented:
📝 Docs preview for commit 3e61988f32284d7b0798391b5655779c031e3180 at: https://e7396ada.fastapitiangolo.pages.dev

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit 73e9e3b2c423069b460d095b235c7610e7aa5c3c at: https://d973e473.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit 7117666fc0f598bccfa0fb045dcb38794657523b at: https://f36b4a0a.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit f6568fda7f5c36f2c380be31876423313b1aa3f7 at: https://5c7ba67e.fastapitiangolo.pages.dev

Commit:
"Kindklasse" -> "Unterklasse"

tiangolo commented:
📝 Docs preview for commit a76dd13ec63f176d31bf8986d2e7f21cc7aacb4a at: https://221f9127.fastapitiangolo.pages.dev

Commit:
"Standard " -> "Standard-"

tiangolo commented:
📝 Docs preview for commit 3be35d9b307e51282392f2049de3c3eee0babff0 at: https://c226d385.fastapitiangolo.pages.dev

Commit:
Forgot some "Antwort" -> "Response"

tiangolo commented:
📝 Docs preview for commit 3fb48dcbf39651f1a73ab198ccf0dfc6449bed13 at: https://0476f9e5.fastapitiangolo.pages.dev

Commit:
"Python " -> "Python-", etc.

tiangolo commented:
📝 Docs preview for commit 630af033197ab4b85c237ba738269dc003e0bd88 at: https://d1445229.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

Commit:
"z.B." -> "z. B."

tiangolo commented:
📝 Docs preview for commit 75ec88dba0ab80f09c2bc05d585931c37eb74459 at: https://9e8ecea4.fastapitiangolo.pages.dev

Commit:
Remove "-"

tiangolo commented:
📝 Docs preview for commit efce289199f4c0b7660c7e1d4bfd8cc1eab36077 at: https://afc4db35.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/path-params.md

tiangolo commented:
📝 Docs preview for commit a1948e39d787263f4fb5682a65ac69cc9474171d at: https://6e8e3427.fastapitiangolo.pages.dev

mattkoehne commented on docs/de/docs/tutorial/path-params.md:
```diff
@@ -0,0 +1,254 @@
+
+### Testen Sie es in der API-Dokumentation
+
+Weil die erlaubten Werte für den *Pfad-Parameter* nun vordefiniert sind, kann die interaktive Dokumentation sie als Auswahl-Drop-Down anzeigen:
```
I guess the common german word is `Dropdown-Menü` instead of `Auswahl-Drop-Down`.

mattkoehne reviewed (changes requested):
Thank you for your work! 🥇

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:

nilslindemann commented:
Thanks a ton @tiangolo for your trust. This simplifies things a lot and helps clear your and my backlog. This comment also goes to all the other merged PRs (I hope you inserted this comment using a script XD)

@mattkoehne Sorry, anyhow I overlooked your review comments in this PR, they were from January :-O. Well, now it's merged. Will later create a PR which applies them. Thanks for your many reviews, including this one.
